### PR TITLE
If command for report the Word comment pressed twice, presents the information in browse mode

### PR DESCRIPTION
**Pull request by tseykovets**
### Link to issue number:

This is a minor change that improves one command for Microsoft Word.

### Summary of the issue:

The text of comments often contains important voluminous information that is difficult to perceive in the mode of pronouncing a full phrase.

This pull request adds the ability to present the text of a comment in browse mode by pressing command twice (NVDA+Alt+C).

### Description of user facing changes

The previously existing command for Microsoft Word (NVDA+Alt+C) gained additional functionality when it is pressed twice.

### Description of development approach

* A handler for the number of presses has been added to the script (if repeats).
* If pressed twice, presents the information in browse mode (ui.browseableMessage).

### Testing strategy:

To test, just press the NVDA+Alt+C command twice on the comment.

The changes do not contain new code to interact with the operating system and Word application.

### Known issues with pull request:

No issues known.
Since the change does not contain new code to interact with the operating system and Word application, it is not expected to affect any other functionality of NVDA.

### Code Review Checklist:

- [x] Documentation:
  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [ ] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] API is compatible with existing add-ons.
- [x] Security precautions taken.

### CODE REVIEW

**Comment by Adriani90** [comment]
The comment editing area in MS Word is not fully accessible, I second this feature. Thanks very much for that. I think a good approach as well would be to think about a nvda specific dialog like in Excel when pressing shift+f2. This would also allow for a more accessible way of editing comments, not only reading them. However, I also understand this is primarily the responsibility of Microsoft to get their part of accessibility done properly.

**Comment by cary-rowen** [comment]
Thank you very much, this is a feature I'm thinking of implementing, good job, please continue.
Please note, that there is a feature request #14628.
You can update the description of this PR.

File: user_docs/en/changes.t2t [hunk 1]
```diff
@@ -10,6 +10,9 @@ What's New in NVDA
 
 
 == Changes ==
+- Microsoft Office:
+  - When command for report the text of the Word comment where the System caret is located (NVDA+Alt+C) pressed twice, presents the information in browse mode.
```
Comment (seanbudd) [inline user_docs/en/changes.t2t:14]:
```suggestion
  - When the command to report the text of the Word comment where the System caret is located is pressed twice  (``NVDA+alt+c``), the information is presented browse mode.
  -
```

File: user_docs/en/userGuide.t2t [hunk 2]
```diff
@@ -1212,7 +1212,7 @@ The Elements List can list headings, links, annotations (which includes comments
 
 +++ Reporting Comments +++[WordReportingComments]
 %kc:beginInclude
-To report any comments at the current caret position, press NVDA+alt+c.
+To report any comments at the current caret position, press NVDA+alt+c. Pressing twice shows the information in a window.
```
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:1215]: Sentences should be on new lines


```suggestion
To report any comments at the current caret position, press ``NVDA+alt+c``.
Pressing twice shows the information in a window.
```

**Review by seanbudd — APPROVED** [review]

**Comment by seanbudd** [comment]
Thanks @tseykovets

**Comment by seanbudd** [comment]
Please note the linting error, repeat is not definied

**Comment by seanbudd** [comment]
Have you tested this? Is it ready for re-review? If so, mark the PR as "ready for review" so it is no longer in draft state.

**Comment by seanbudd** [comment]
have you tested this? please do not mark as ready for review if you have not tested this.

**Review by Qchristensen — APPROVED** [review]
As with #15986, this will be a good feature for some users.  The user guide reads well, and again I would make the changes.txt wording similar to my suggestion on the other PR:

- Word: When "report any comments at the current caret position"  (``NVDA+alt+c``) is pressed twice, the information is shown in a window. (#15987, @tseykovets)

**Comment by Adriani90** [comment]
@tseykovets are you still working on this?

**Comment by tseykovets** [comment]
@Adriani90, it seems to me that I have implemented everything that is needed.
I would appreciate testing and information if something is wrong.

**Comment by Adriani90** [comment]
@tseykovets you need to resolve the conflicts with the current master otherwise we cannot download any build to test.
But is it very difficult for you to test this code yourself first?

**Comment by Adriani90** [comment]
@tseykovets are you still working on this? Or should we mark this PR as abandoned?

**Comment by cary-rowen** [comment]
If you are unable to continue with this I would be happy to take over

**Comment by tseykovets** [comment]
At the end of 2023, I considered the implementation from this pull request to be complete and debugged.
However, I don't know how true this is for the current state of the master branch.
At the moment, I do not have the opportunity to quickly test the implementation, since I do not have a computer at hand with a configured environment for the local build of NVDA.
If someone undertakes to check the relevance of the implementation from this pull request, then I will be grateful for such help.

**Comment by seanbudd** [comment]
You can find the build created by the pull request in the PR check for appveyor.
If you find the status check (near the close button, or last commit)  it will include a link to the appveyor build.
You can use the build artifact for the installer to test.
I'd encourage you in future to please consider creating a test environment before contributing.
We cannot accept code changes without testing from the author.

**Comment by gerald-hartig** [comment]
As @seanbudd mentioned, PRs can only move from draft to ready status once testing has been successfully completed. Thorough testing is crucial to maintain the integrity and functionality of NVDA and is part of our Contribution Process (https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/contributing.md#overview-of-contribution-process). We strongly advise authors to test their own code but if the author is unable to do the testing, collaboration with someone in the community to carry out that testing is then essential. 

If for some reason before you start you know that you cannot carry out the testing, we advise you find someone who can do the testing before you write your first line of code. You can use the issue ticket for this purpose, or reach out to the development community in one of the user/developer groups.

For this PR, it looks like @cary-rowen may be able to assist with the testing. Otherwise we may need to consider closing this PR until testing can be completed.

**Comment by seanbudd** [comment]
Closing as abandoned

**Pull request closed without merging**